app/Http/Controllers/Admin/DigitalPublicationArticleController.php

```diff
@@ -2,10 +2,11 @@
 
 namespace App\Http\Controllers\Admin;
 
+use A17\Twill\Http\Controllers\Admin\NestedModuleController;
 use App\Repositories\DigitalPublicationRepository;
 use App\Http\Controllers\Admin\Behaviors\IsNestedModule;
 
-class DigitalPublicationArticleController extends ModuleController
+class DigitalPublicationArticleController extends NestedModuleController
```

Comment:
The `ModuleController` this used to extend included all the autorelated stuff. I'm not sure we'll need that here? But flagging it just in case.

Reply:
@nikhiltri just from quick testing it doesn't seem like we pass auto-related or related content to the articles